torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2306,6 +2306,44 @@ def generator():
 
     return list(generator())
 
+
+def sample_inputs_conv2d(op_info, device, dtype, requires_grad, jit_fail_sample=False, **kwargs):
```

Comment:
It's nice to be able to test some of the cudnn=False cases. Do you have ideas on how we can do that or if that's possible using OpInfo? The mechanism for testing the cudnn=False case is to set `torch.backends.cudnn.enabled = False` while running the test

Reply:
Have added a new entry with wrapper which turns cudnn off. Let me know what you feel about it.

Reply:
On second thought, I'm not sure that OpInfo should be in the business of testing with cudnn turned off. The rationale is that this seems very specific to the CUDA backend and that most users won't run into this. What are your thoughts, @mruberry @ngimel?

Reply:
Let me know if we should remove this entry for now (and pursue it later if needed?) Thanks!